documentation/assemblies/upgrading/assembly-downgrade.adoc

```diff
@@ -0,0 +1,26 @@
+// This assembly is included in the following assemblies:
+//
+// deploying/master.adoc
+
+[id='assembly-downgrade-{context}']
+= Downgrading Strimzi
+
+If you are encountering issues with the latest version of Strimzi,
+you can revert your installation to the previous version.
+
+As with the Strimzi upgrade process, you can perform your downgrade in two stages:
+
+. Revert your Cluster Operator to the previous Strimzi version.
+** xref:proc-downgrade-cluster-operator-{context}[]
+
+. Downgrade all Kafka brokers and client applications to the previous Kafka version.
+** xref:assembly-downgrade-kafka-versions-{context}[]
```

Comment:
I think this will be confusing ... saying it happens in two stages for me gives it a particular order. But the order is not so simple. For example, I'm running Strimzi 0.19.0 with Kafka 2.5. And I upgrade first to Strimzi 0.20 + Kafka 2.5 and then to Kafka 2.6. In that case, the order here is actually first to downgrade Kafka 2.6 to 2.5 if still possible and then to downgrade Strimzi 0.20 to 0.19.

Reply:
I've simplified to:

```
You can perform a downgrade to:

. Revert your Cluster Operator to the previous Strimzi version.
** xref:proc-downgrade-cluster-operator-{context}[]

. Downgrade all Kafka brokers and client applications to the previous Kafka version.
** xref:assembly-downgrade-kafka-versions-{context}[]
```

Reply:
This section won't be in the doc yet